gita/utils.py

```diff
@@ -421,13 +421,14 @@ def describe(repos: Dict[str, Dict[str, str]], no_colors: bool = False) -> str:
     Return the status of all repos
     """
     if repos:
+        truncator = info.Truncate()
```

Comment:
We instantiate the Truncate class here so all threads will see the same values and the file only needs to be read once.